### ISSUE TITLE
After offline bootstrap, `eb` returns error unless EASYBUILD_BOOTSTRAP_SOURCEPATH is unset

### ISSUE THREAD

**Issue by runiq** [easybuilders/easybuild-framework#2934]
Moved from https://github.com/easybuilders/easybuild/issues/519.

I was trying the offline bootstrapping procedure described [here](https://easybuild.readthedocs.io/en/latest/Installation.html#offline-bootstrapping-using-supplied-source-tarballs) which completed successfully. However, after loading the resulting `EasyBuild/3.9.2` module, every invocation with `eb` exits with the following error:

```
$ eb --help
ERROR: Failed to parse configuration options: 'Found 1 environment variable(s) that are prefixed with EASYBUILD but do not match valid option(s): EASYBUILD_BOOTSTRAP_SOURCEPATH'
```

The error goes away when I `unset EASYBUILD_BOOTSTRAP_SOURCEPATH`.

**Comment by boegel** [comment]
Yeah, that's sort of expected... It's mainly due to an unfortunate choice in the bootstrap script to go with `EASYBUILD_BOOTSTRAP_SOURCEPATH`.

Any environment variable that starts with `EASYBUILD_` is picked up by EasyBuild, and it'll try and figure out the corresponding configuration option for those. If it can't find a match, it will complain like you noticed.

The only way to fix this would be deprecating the use of `$EASYBUILD_BOOTSTRAP_SOURCEPATH` by the bootstrap script, and start using `$EB_BOOTSTRAP_SOURCEPATH` instead for example (which won't be picked up by EasyBuild itself).

**Comment by Flamefire** [comment]
How about not using env-variables in the bootstrap script but cmd line options?
